extensions/data-transfer/portability-data-transfer-google/src/main/java/org/datatransferproject/datatransfer/google/photos/GooglePhotosExporter.java

```diff
@@ -342,21 +347,41 @@ private List<PhotoModel> convertPhotosList(
     }
 
     for (GoogleMediaItem mediaItem : mediaItems) {
-      if (mediaItem.getMediaMetadata().getPhoto() != null) {
-        // TODO: address videos
-        boolean shouldUpload = albumId.isPresent();
-
-        if (tempMediaData != null) {
-          shouldUpload = shouldUpload || !tempMediaData.isContainedPhotoId(mediaItem.getId());
-        }
+      // TODO: address videos
+      boolean shouldUpload = albumId.isPresent();
+      if (tempMediaData != null) {
+        shouldUpload = shouldUpload || !tempMediaData.isContainedPhotoId(mediaItem.getId());
+      }
 
-        if (shouldUpload) {
-          PhotoModel photoModel = GoogleMediaItem.convertToPhotoModel(albumId, mediaItem);
-          photos.add(photoModel);
+      if (!shouldUpload) {
```

Comment:
prefer to do

if (shouldUpload) {
} else {
}

Reply:
is it better now? 

if !shouldUpload:
   throw new Exception